config/_default/menu.yaml

```diff
@@ -1,10 +1,12 @@
 main:
-- identifier: home
-  name: Home
+- name: Home
```

Comment:
The title link on the left of the navbar also points to the home page. Therefore is it required to give another home link?

Reply:
It is a matter of personal preference. I like to have a home page just like [my website](https://amirpourmand.ir/). 

We can comment this or make it by default. I leave it to you.